jwst/cube_build/src/cube_match_internal.c

```diff
@@ -0,0 +1,484 @@
+/*
+The detector pixels are represented by a 'point could' on the sky. The IFU cube is
+represented by a 3-D regular grid. This module finds the point cloud members contained
+in a region centered on the center of the cube spaxel. The size of the spaxel is spatial
+coordinates is cdetl1 and cdelt2, while the wavelength size is zcdelt3.
```

Comment:
typo? should "zcdelt3" be just "cdelt3"?

Reply:
Oh, and "cdetl1" should be "cdelt1"

Reply:
fixed